### PR TITLE
Add extra section for volumes in the quick start docs

### PR DESCRIPTION
**Pull request by cristobal23**
This pull request is meant to address issue #1154

### CODE REVIEW

**Comment by aanand** [comment]
Thanks! (ping @fredlf @SvenDowideit)

- Commits need signing off.
- This might need rebasing once https://github.com/docker/compose/pull/1162 is merged.

If you don't know how to do either of those, I'll happily merge this manually once we've got an LGTM from the docs team.

**Comment by SvenDowideit** [comment]
ya, LGTM :)

File: docs/index.md [hunk 1]
```diff
@@ -185,6 +181,51 @@ your services once you've finished with them:
...
+def hello():
+    redis.incr('hits')
+    return 'Hello World! I have been seen %s times.' % redis.get('hits')
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", debug=True)
+```
+
+You will notice that the changes are not reflected in the browser.  Instead of running `docker-compose stop`
+`docker-compose build` and `docker-compose up` to recreate the container every time you make a change, you can add a volume instead. In order to mount the current directory inside the container so we can work on code without having to rebuild the image, modify `docker-compose.yml`:
```
Comment (fredlf) [inline docs/index.md]: Please wrap line at 80 chars. For consistency, please change "we" to "you".

File: docs/index.md [hunk 2]
```diff
@@ -185,6 +181,51 @@ your services once you've finished with them:
...
+      ports:
+       - "5000:5000"
+      volumes:
+       - .:/code
+      links:
+       - redis
+    redis:
+      image: redis
+
+Recreate the container and make some more changes. If you are not running in -d mode you will see:
```
Comment (fredlf) [inline docs/index.md]: Code format for `-d` please.

**Comment by fredlf** [comment]
A couple of copy-edit nit picks, but otherwise LGTM. Thanks for the contribution!

**Comment by aanand** [comment]
@cristobal23 Could you rebase and implement @fredlf's changes please?

**Comment by dnephin** [comment]
Closing since there hasn't been any response in a while.

Since volumes are changing soon, we'll probably have to change our volume documentation accordingly.

**Pull request closed without merging**